webservice/components/webservice_component.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import requests
+
+from odoo.addons.component.core import AbstractComponent, Component
+
+
+class BaseWebServiceAdapter(AbstractComponent):
+    _name = "base.webservice.adapter"
+    _collection = "webservice.backend"
+
+
+class BaseRestRequestsAdapter(Component):
+    _name = "base.requests"
+    _usage = "webservice.httprequest"
+    _inherit = "base.webservice.adapter"
```

Comment:
so, by looking at https://github.com/OCA/edi/pull/295 you should be able to define this like

```suggestion
class BaseRestRequestsAdapter(Component):
    _name = "base.requests"
    _usage = "webservice.request"
    _webservice_protocol = "http"
    _inherit = "base.webservice.adapter"
```